src/repository/pcapi/__specs__/pcapi.spec.js

```diff
@@ -373,14 +373,17 @@ describe('pcapi', () => {
       // Given
       const filters = {
         venueId: 'AA',
+        eventDate: new Date(2020, 8, 13),
         page: 2,
       }
 
       // When
       await loadFilteredBookingsRecap(filters)
 
       // Then
-      expect(client.get).toHaveBeenCalledWith('/bookings/pro?page=2&venueId=AA')
+      expect(client.get).toHaveBeenCalledWith(
+        '/bookings/pro?page=2&venueId=AA&eventDate=2020-09-13T00%3A00%3A00Z'
```

Comment:
```suggestion
        '/bookings/pro?page=2&venueId=AA&eventDate=2020-09-13'
```

On peut peut être éviter d'appeler l'API avec un timestamp vu que ce n'est pas nécessaire ?

Reply:
J'ai préféré rester sur un format ISO standard dans l'appel, et comme ça on laisse l'API gérer ensuite